airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -809,10 +810,13 @@ class EmrServerlessCreateApplicationOperator(BaseOperator):
       Its value must be unique for each request.
     :param config: Optional dictionary for arbitrary parameters to the boto API create_application call.
     :param aws_conn_id: AWS connection to use
-    :param waiter_countdown: Total amount of time, in seconds, the operator will wait for
+    :param waiter_countdown: (deprecated) Total amount of time, in seconds, the operator will wait for
         the application to start. Defaults to 25 minutes.
-    :param waiter_check_interval_seconds: Number of seconds between polling the state of the application.
-        Defaults to 60 seconds.
+    :param waiter_check_interval_seconds: (deprecated) Number of seconds between polling the state
```

Comment:
Not super fan of making user change their code for no reason except to standardize names.
This is adding friction to upgrading where we could very well keep the current name since there is no behavior change.

Reply:
I think consistent naming is a worthy effort because it leads to more maintainable code. In this case, there is no reason for the user to change their code at this point. The way it is written, if a user does not specify any waiter related parameters, they will not see any difference, including the deprecation message. Only if the user is specifically using the old parameters will the message be shown and they will not see any difference in behavior - the new waiter parameters are computed from the old ones. I think its important to update variable names as code changes, otherwise it will lead to a disconnect between variable names, and what they represent.